build.py

```diff
@@ -20,7 +20,7 @@ def change_secret_key(keygen):
     if file_string[-1] != "\n":
         file_string += "\n"
     if "DJANGO_SECRET_KEY" not in file_string:
-        file_string += "DJANGO_SECRET_KEY=" + keygen
+        file_string += "DJANGO_SECRET_KEY={}".format(keygen)
```

Comment:
this is not an f string, right?

Reply:
@nemesisdesign yes that's right. this is the `.format()` function so it can also work on other Python versions such as Python 2.7 - 3.4. etc.

Reply:
@nemesisdesign yes, the `build.py` runs during build on host machine and the user can have any python version! :smile:

Reply:
ok, I'd add a comment then to make it clear, eg: `# no f-string because this is run on host machine, user may not have python 3.7`.